letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -1308,7 +1342,8 @@ def get_version(self):
 
         """
         try:
-            stdout, _ = le_util.run_script([self.conf("ctl"), "-v"])
+            stdout, _ = le_util.run_script(
+                constants.os_constant("version_cmd").split(" "))
```

Comment:
Parsing the command in this way is slightly hackish; it might create headaches for us if we ever want to support paths with spaces in them.  But I'm not sure that I have a better idea (we don't really want to make people type python lists on the command line); perhaps if anyone ever asks for that, we can add some kind of escaping to this parser line?